reddit_decider/__init__.py

```diff
@@ -437,6 +437,12 @@ def expose(
             to :code:`events_logger` (keys must be part of v2 event schema,
             use dicts for nested fields) under :code:`inputs` and as :code:`kwargs`
         """
+        if variant_name is None or variant_name == "":
+            logger.info(
+                f"`variant_name` arg not provided in reddit_decider.expose() call for experiment: {experiment_name}"
+            )
```

Comment:
debating if we should remove this log altogether, since it's bad practice to expose variant-less events, but it also doesn't really hurt much either.

Or maybe make it `debug` logger?

cc @gshipilov

Reply:
Well, do we consider this case to be an error?

It looks like below we're logging a returned error at a `warn` level. It might make sense to do the same here.

Given that this method is effectively deprecated I'm also ok with just not logging anything.

Reply:
it's not an explicit "error" of things going wrong, but just an anti-pattern, this function shouldn't be called without a `variant_name` since it puts load on the event collector but isn't technically breaking anything.

Reply:
ended up removing it to play it safe and not introduce new loggings